src/Step.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import Button from './button'
+
+class Step extends Component {
```

Comment:
Ran out of time but next step is having the step from the state activate the current step...we can discuss that!